boards/nxp/mcx_nxxx_evk/mcx_n5xx_evk_mcxn547_cpu0.yaml

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright 2025 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+identifier: mcx_n5xx_evk/mcxn547/cpu0
+name: NXP MCX-N5XX-EVK (CPU0)
+type: mcu
+arch: arm
+ram: 320
+flash: 2048
+toolchain:
+  - zephyr
+  - gnuarmemb
```

Comment:
Are'nt there more features enabled in this board that should be added here?

Reply:
hake said to remove them in a comment above